source/coding-the-legislation/35_periods.md

```diff
@@ -25,12 +45,12 @@ In OpenFisca inputs, periods are encoded in strings. All the valid period format
 | `year:AAAA:N`     | N years         | `'year:2010:3'`     | The years 2010, 2011 and 2012.                   | From the 1st of January 2010 to the 31st of December 2012, inclusive. |
 | `year:AAAA-MM:N`  | N rolling years | `'year:2010-04:3'`  | The three years period starting in April 2010.   | From the 1st of April 2010 to the 31st of March 2013, inclusive.      |
 | `month:AAAA-MM:N` | N months        | `'month:2010-04:3'` | The three months from April to June 2010.        | From the 1st of April 2010 to the 30th of June 2010, inclusive.       |
+| `day:AAAA-MM-DD:N` | N days        | `'day:2010-04-01:30'` | The thirty days of April 2010. | From the 1st of April 2010 to the 30th of April 2010, inclusive.       |
```

Comment:
It's also not clear to me why would someone use `'day:2010-04-01:30'` instead of `'2010-04'`. Could we replace by `'day:2010-04-01:15'` and describe it as "The first 15 days of April"?